From 857719e82daea0d85b734cac34cf569050724068 Mon Sep 17 00:00:00 2001
From: Tim Orling <tim.orling@konsulko.com>
Date: Sun, 20 Feb 2022 20:26:51 -0800
Subject: [PATCH] setup.py: use vendored _distutils

Deprecation warning of distutils is interferring with bdist_wheel build.

For now, use the vendored setuptools._distutils.core.

Upstream-Status: Pending [upstream appears unmaintained]

Signed-off-by: Tim Orling <tim.orling@konsulko.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 1bf87cf..a49fb8e 100644
--- a/setup.py
+++ b/setup.py
@@ -1,5 +1,5 @@
 import os.path
-from distutils.core import setup
+from setuptools._distutils.core import setup
 
 readme_file = os.path.join(os.path.dirname(__file__), 'README.md')
 readme = open(readme_file).read()
